templates/about.html

```diff
@@ -4,7 +4,8 @@
       <p style="margin-top: 6px;">
 
         Everyone loves the&nbsp;<a target="_blank" href="https://developers.google.com/speed/libraries/?csw=1">Google
-        CDN</a>&nbsp;right? Even&nbsp;<a href="https://www.asp.net/ajax/cdn" target="_blank">Microsoft runs a public CDN</a>.
+        CDN</a>&nbsp;right? Even&nbsp;<a href="https://www.asp.net/ajax/cdn" target="_blank">Microsoft runs a public
+        CDN</a>.
```

Comment:
this is not related to the commit topic

Reply:
This is due to the reformat that I performed on this page before I made major modifications to ensure code quality was maintained in my edits.

Reply:
This is still not fixed. It doesn't make sense to be contained in this commit:
```
commit c4fc4a8dc4ca88315df7cf9aa2a071424ee59bdf
Author: MattIPv4 <matthew@cowley.org.uk>
Date:   Wed Jan 2 20:29:07 2019 +0000

    add MattIPv4 to about
```

Reply:
Unsure why is it within this commit, will move it to the refactor commit.